app/mailers/concerns/addresses.rb

```diff
@@ -0,0 +1,15 @@
+module Addresses
+  include ActiveSupport::Concern
+
+  def prison_mailbox_email
+    @visit.prison_email
```

Comment:
I'm confused by the naming here. What does 'mailbox' mean? Why is it not `prison_email_address`?